CodeEditor/Sources/CodeEditor/UXCodeTextViewRepresentable.swift

```diff
@@ -152,8 +152,10 @@ public struct UXCodeTextViewRepresentable: UXViewRepresentable {
         
         public func textView(_ textView: UITextView, shouldInteractWith URL: URL,
                              in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
-            if interaction == .invokeDefaultAction {
-                self.parent.editorBindings.feedbackForSelectionId.wrappedValue = URL.absoluteString
+            if interaction == .invokeDefaultAction,
+               let feedbackId = UUID(uuidString: URL.absoluteString)
```

Comment:
Why is this declaration in the `if` condition part? Can't it be in the block?

Reply:
`UUID(uuidString:)` returns an optional. It should normally never be nil, since all highlights are linked to a feedback id. However, if it somehow does become nil, the block should not be executed. That's why it's a part of the if condition.